### ISSUE TITLE
loss is Nan

### ISSUE THREAD

**Issue by hh123445** [open-mmlab/mmrotate#326]
When I use KFIoU to train rotate Retinanet,after 30epoch there have error for loss

FutureWarning: Non-finite norm encountered in torch.nn.utils.clip_grad_norm_; continuing anyway. Note that the default behavior will change in a future release to error out if a non-finite total norm is encountered. At that point, setting error_if_nonfinite=false will be required to retain the old behavior.return clip_grad.clip_grad_norm_(params, **self.grad_clip)

lr: 2.499e-03, eta: 25 days, 15:10:41, time: 2.080, data_time: 0.067, memory: 4759, loss_cls: nan, loss_bbox: nan, loss: nan, grad_norm: na
![2](https://user-images.githubusercontent.com/90760124/170283035-c0647927-b54c-4810-88af-87a50daff891.png)
n

**Comment by zytx121** [comment]
How many GPU do you use?  KFIoU is not stable.

**Comment by hh123445** [comment]
> 您使用多少 GPU？KFIoU 不稳定。

My setting is 
    samples_per_gpu=4
    workers_per_gpu=8

**Comment by zytx121** [comment]
Maybe you can try to use samples_per_gpu=2.

**Comment by hh123445** [comment]
> Maybe you can try to use samples_per_gpu=2.
Thanks for your answer,I'll have a try.
